docs/ovs-pipeline.md

```diff
@@ -581,51 +579,48 @@ non-ARP and non-IP (assuming any can be received by the switch) is actually
 dropped much earlier in the pipeline ([SpoofGuardTable]). In the future, we may
 need to support more cases for L2 multicast / broadcast traffic.
 
-### CnpIngressRuleTables (85-89)
+### AntreaPolicyIngressRuleTables (85-89)
 
-These tables are very similar to [CnpEgressRuleTables], but implement the ingress
-rules of the CNP. Depending on the tier to which the CNP belongs to, the rules will
-be installed in a table corresponding to that tier. The ingress CNP table to tier
-mappings is as follows:
+These two tables are very similar to [AntreaPolicyEgressRuleTables], but implement
+the ingress rules of Antrea Policies. Depending on the tier to which the policy belongs
+to, the rules will be installed in a table corresponding to that tier.
+The ingress table to tier mappings is as follows:
 
 ```
-1. Emergency tier -> EmergencyIngressRuleTable(85)
-2. SecurityOps tier -> SecurityOpsIngressRuleTable(86)
-3. NetworkOps tier -> NetworkOpsIngressRuleTable(87)
-4. Platform tier -> PlatformIngressRuleTable(88)
-5. Application tier -> ApplicationIngressRuleTable(89)
+Application tier -> DefaultTierIngressRuleTable(89)
+All other tiers  -> MultiTierIngressRuleTable(85)
 ```
 
-Again for this table, you will need to keep in mind the CNP
+Again for these two tables, you will need to keep in mind the Antrea Policy
 [specification](#antrea-network-policy-crd-implementation) that we are using.
 Since the example CNP resides in the Application tier, if you dump the flows
```

Comment:
maybe also use ACNP instead of CNP in this doc. As we moved our short name to acnp

Reply:
Done